src/Gui/CommandStd.cpp

```diff
@@ -1042,6 +1043,40 @@ void StdCmdAnnotationLabel::activated(int)
     commitCommand();
 }
 
+//===========================================================================
+// Std_CommandPalette
+//===========================================================================
+DEF_STD_CMD_A(StdCmdCommandPalette)
+
+StdCmdCommandPalette::StdCmdCommandPalette()
+    : Command("Std_CommandPalette")
+{
+    sGroup = "Tools";
+    sMenuText = QT_TR_NOOP("Command &Palette...");
+    sToolTipText = QT_TR_NOOP("Open the command palette to search and execute commands");
+    sWhatsThis = "Std_CommandPalette";
+    sStatusTip = sToolTipText;
+    sAccel = "Ctrl+Shift+P";

```

Comment:
For me that shortcut does not work reliably - and for thing like that it must work 100% of times